### DISCUSSION TITLE
Requiring photo types

### DISCUSSION THREAD

**Discussion by rjheby** [PullDating/Org#109] [Ideas & Features]
Users should be required to have at least these two photos:

One of the face, unobscured, basically head on. No sunglasses, no filters, no coverings. It should show what the face looks like.

A second picture showing the person's body, if they are able to, standing. Their full body should be in the frame but not super zoomed out. This is to show physique. Doesn't have to be bathing suit or anything like that.

Other thoughts on photos: limiting number of photos with friends, limiting the filters, zooms, angles. Pictures should be a visual representation of a person.

**Comment by Oshyan** [comment]
This would be ideal, but could be challenging to enforce. That said it would be great if we could. One idea I had which might make it easier if we were going to use image analysis algos to do is: we have people select a "type" or "subject" for each photo they upload, e.g. "face", "profile/body", "activity", "group", etc. That way it's easier to check the photo because we have a narrower range of what it should be, i.e. we can test for "body vs. anything but body" rather than having to try to evaluate "body, face, activity, group?". They tell *us*. Then we can more easily test of they were honest (still non-trivial, but easier). We could also treat each type of photo differently if desired, e.g. for group shots we require that they crop to their head, we'll show the full photo but highlight their face with a click/tap or something.

Not only that but we could also let people browse with photos by face, vs. body, etc. Like let's say someone just wants to see body shots in the gallery view or whatever. Of course that prevents people from using their preferred photo as the main photo perhaps, but I have thoughts about that too, maybe it could just be an override for that. And anyway if all people will actually do is click into the profile and scan for the body shot *anyway*, why not let them and just make it easier? I think trying to enforce good behavior has its benefits, but should also have its limits. We want to enable people to get what they want out of the platform, *as long as what they want is not hurting someone else*.

**Reply by rjheby** [reply]
I like the idea of self identifying which type of picture you're uploading. 

Can't we also leverage the community for moderating the pics? E.g. they can tag or report a photo as mislabeled. It's not gonna be taken down just needs a new label. Then that report can go back to the original user and say something like "hey userxyz thinks this is a body pic, want to relabel it?" If there are multiple reports then it will get flagged to actual moderators.

Also we should figure out a strict guideline for posting crude content. And definitely be pretty strict on posting cartoons or memes or other dumb things that aren't an actual person. 

Plus there should be repurcussions for false or revenge reports.

**Comment by Oshyan** [comment]
Yeah, I think you covered the bases here. User reporting is good and we should definitely act on those reports, but we also need to track reporters and make sure *they're* being honest, hah. So we have bad actors for incorrect profile pics, we have bad actors of people falsely reporting, etc. Lots of stuff to monitor and deal with. I agree though that having a good user report system and way of handling auto-moderation, etc. with manual review if needed is valuable. I think the Discourse "trust level" system might be an interesting thing to look at for reference, not necessarily in the "levels" (although that brings up its own interesting idea), but at least the *criteria* they use to e.g. take automated action for moderation, etc.

**Comment by ajbott** [comment]
The photo types is a good feature for collecting data on what photos work best in profiles that get the most matches. I don't think this is a critical feature that would be included anytime soon due to it's impact on match performance. I think the community guidelines on photo posting would best cover what photos not to post (memes, cartoons, NSFW, ect), but I believe restricting how people want to setup their profile is limiting in terms of our research. I think we should be able to offer recommendations based on our research, but never hinder. specific community guidelines (depending on which community they join) should be the only restrictive factor when setting up their profile images.
